modules/installation/pages/pubcloud-requirements.adoc

```diff
@@ -51,17 +51,17 @@ To access the {productname} {webui}, allow HTTPS when configuring the network ac
 
 == Separate Storage Volumes
 
-We recommend that the repositories and the database for {productname} are stored on a separate storage device.
+We recommend that the repositories and the database for {productname} are stored on separate storage devices.
 This will help to avoid data loss in cases when the {productname} instance is terminated.
```

Comment:
I think terminating the instance is not the issue according to the problem description of this issue.

Reply:
It could be an issue, as terminating an instance deletes the root volume. An external volume helps protecting the data against loss due to instance termination (intentionally or not). But I agree that it's not the only and most probably not the most important reason to use separate storage volumes for data. Do you think we need to expand on this?